components/Paths/PathItem.tsx

```diff
@@ -1,14 +1,26 @@
 'use client';
+
+import classNames from 'classnames';
+
 interface PathProps {
   item: {
     title: string;
     description: string;
     link: { name: string; href: string };
-    btnColorDark: string;
+    btnColor: string;
   };
 }
 
 export default function PathItem({ item }: PathProps) {
+  const buttonClasses = classNames(
+    'text-base font-normal font-inter px-4 py-2 body-bold w-fit rounded-xl md:bg-lightText md:dark:bg-gray-700',
+    item.btnColor,
+    {
+      [`hover:${item.btnColor}`]: true,
+      [`hover:dark:${item.btnColor}`]: true,
+    }
```

Comment:
If you know that those classes will always get `true` then no need to use object just -> 
```suggestion
    `hover:${item.btnColor} hover:dark:${item.btnColor}`
```